HST Office packaging: installer product definition.

The EPM/MSI installer looks up the product by PRODUCTNAME without spaces
("HSTOffice"), which upstream's openoffice.lst.in does not define, so
`make` failed with "Product HSTOffice not defined". Adds an HSTOffice
section (copy of the LibreOffice one) without the TDF update URL and with
HSTOffice_* download names. Profile dir stays product-specific
(~/.config/hst office/4 from instsetoo_native/CustomTarget_setup.mk).

License: modified upstream file keeps its license (MPL-2.0).
---
--- a/instsetoo_native/util/openoffice.lst.in
+++ b/instsetoo_native/util/openoffice.lst.in
@@ -73,6 +73,41 @@
     }
 }
 
+HSTOffice
+{
+    Settings
+    {
+        variables
+        {
+            PRODUCTNAME @PRODUCTNAME@
+            PRODUCTVERSION @LIBO_VERSION_MAJOR@.@LIBO_VERSION_MINOR@
+            PRODUCTEXTENSION .@LIBO_VERSION_MICRO@.@LIBO_VERSION_PATCH@@LIBO_VERSION_SUFFIX@
+            POSTVERSIONEXTENSION
+            BUNDLEIDENTIFIER @MACOSX_BUNDLE_IDENTIFIER@
+            BRANDPACKAGEVERSION @LIBO_VERSION_MAJOR@.@LIBO_VERSION_MINOR@
+            USERDIRPRODUCTVERSION 4
+            BASEPRODUCTVERSION @LIBO_VERSION_MAJOR@.@LIBO_VERSION_MINOR@
+            ADD_INCLUDE_FILES cli_ure/version/version.txt,unoil/climaker/version.txt
+            ADDSYSTEMINTEGRATION 1
+            PACKAGEVERSION @LIBO_VERSION_MAJOR@.@LIBO_VERSION_MINOR@.@LIBO_VERSION_MICRO@.@LIBO_VERSION_PATCH@@LIBO_VERSION_SUFFIX@
+            PACKAGEREVISION {buildid}
+            LICENSENAME @LICENSE@
+            ROOTMODULEGID gid_Module_Root
+            GLOBALPATCHFILEGID gid_File_Txt_Patchfiles
+            SPELLCHECKERFILE spellchecker_selection.txt
+            CHANGETARGETDIR 1
+            PATCHCODEFILE ooo_patchcodes.txt
+        }
+        active          1
+        compression     5
+        script          setup_osl
+        downloadname    HSTOffice_{productversion}_{os}_install_{languages}
+        langpackdownloadname    HSTOffice_{productversion}_languagepack_{os}_install_{languages}
+        helppackdownloadname    HSTOffice_{productversion}_helppack_{os}_install_{languages}
+        include         {installscriptdir},{customtargetpath}/readlicense_oo/readme,{customtargetpath}/sysui/{pkgtype},{customtargetpath}/setup_native,{filelistpath}/PackageSet,{filelistpath}/GeneratedPackage,{filelistpath}/Package,{filelistpath}/ExtensionPackage,{filelistpath}/CustomTarget/helpcontent2/help3xsl/filelists,{extensionsdir},{licensepath},{packinfopath}
+    }
+}
+
 LibreOfficeDev
 {
     Settings
